graphql/src/main/java/io/micronaut/configuration/graphql/ws/apollo/GraphQLApolloWsResponse.java

```diff
@@ -32,12 +33,18 @@
 @Serdeable
 public class GraphQLApolloWsResponse {
 
-    private final String type;
+    private String type;
```

Comment:
make a creator instead of exposing these properties as writable

Reply:
@guillermocalvo I think by "make a creator" he means annotating the main constructor with (`@Creator`).

Reply:
Done 8bc929934887262fb65f0f5910216d934809fe3f